### PR TITLE
fix(#270): fix wrong action sheet ref

### PR DESCRIPTION
**Pull request by giautm**
# Submit a pull request

Fix: https://github.com/GetStream/stream-chat-react-native/issues/270

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: src/components/MessageInput/MessageInput.js [hunk 1]
```diff
@@ -873,7 +875,7 @@ class MessageInput extends PureComponent {
           */}
 
         <ActionSheetAttachment
-          setAttachActionSheetRef={this.setInputBoxRef}
+          setInputBoxRef={this.setAttachActionSheetRef}
```
Comment (vishalnarkhede) [inline src/components/MessageInput/MessageInput.js:878]:
```suggestion
          setAttachActionSheetRef={this.setAttachActionSheetRef}
```

Comment (vishalnarkhede) [reply src/components/MessageInput/MessageInput.js:878]: Also you will need to change the prop in ActionSheetAttachment component from `setInputBoxRef` to `setAttachActionSheetRef`

Comment (vishalnarkhede) [reply src/components/MessageInput/MessageInput.js:878]: Also make sure to change the typescript declaration file as well :)

**Comment by vishalnarkhede** [comment]
Fixed in https://github.com/GetStream/stream-chat-react-native/pull/273

**Comment by giautm** [comment]
I don't know, when I can break your API. So, I just use the old property was defined. Thank you for your fix.

**Pull request closed without merging**